puppet/modules/slave/manifests/init.pp

```diff
@@ -193,6 +195,15 @@
   # nodejs/npm for Katello JavaScript tests
   # packages only really available on EL6+
   if $::osfamily == 'RedHat' {
+    yumrepo { 'google-chrome':
+      name => 'google-chrome - \$basearch',
+      ensure => present,
+      baseurl => 'http://dl.google.com/linux/chrome/rpm/stable/\$basearch',
+      enabled => '1',
+      gpgcheck => '1',
+      gpgkey => 'https://dl-ssl.google.com/linux/linux_signing_key.pub',
+      before => Package['google-chrome-stable'],
```

Comment:
alignment is off here

Reply:
@ekohl I've updated the PR to add a space after the `yumrepo` resource, don't see any alignment issues here, indentation is the same as the other keys, could you clarify?

Reply:
Looks like we don't have lint enabled but we align the arrows. See the file declaration above it.